src/Workspaces/Core/Portable/CodeStyle/CodeStyleOption.cs

```diff
@@ -7,7 +7,11 @@
 using System.Xml.Linq;
 using Microsoft.CodeAnalysis.Diagnostics;
 
+#if CODE_STYLE
+namespace Microsoft.CodeAnalysis.Internal.Options
```

Comment:
Namespace changes for all options related types are needed to avoid use of extern aliases in CodeFixes and UniTests project which have access to both the core CodeStyle project and Workspaces project where they are defined.